### PR TITLE
Add session post

### PR DESCRIPTION
**Pull request by jebbinSCL**

### CODE REVIEW

**Comment by dmorrisSL** [comment]
:shipit:

File: Gateway/DataTransform.fs [hunk 1]
```diff
@@ -54,6 +54,16 @@ module Session =
...
 
+    let toDto (session : Models.Session) : Dtos.Session = 
+        { Id = session.Id
+          Title = session.Title
+          Description = session.Description
+          Status = session.Status
+          Date = session.Date
+          DateAdded = session.DateAdded
+          SpeakerId = session.Speaker.Id
+          AdminId = session.Admin |> Option.map (fun admin -> admin.Id) }
```
Comment (pallenSL) [inline Gateway/DataTransform.fs]: Why isn't this session.Admin.Id ?

Comment (jebbinSCL) [inline Gateway/DataTransform.fs]: The admin is optional.

**Comment by pallenSL** [comment]
:shipit:

**Pull request merged**